libs/wingsdk/src/target-sim/container.ts

```diff
@@ -0,0 +1,144 @@
+import { Construct } from "constructs";
+import { ISimulatorResource } from "./resource";
+import { ContainerSchema } from "./schema-resources";
+import { simulatorAttrToken } from "./tokens";
+import { bindSimulatorResource, makeSimulatorJsClient } from "./util";
+import { fqnForType } from "../constants";
+import { App } from "../core";
+import { INFLIGHT_SYMBOL } from "../core/types";
+import { Util as fs } from "../fs";
+import { isPath } from "../shared/misc";
+import { BaseResourceSchema } from "../simulator/simulator";
+import { IInflightHost, Resource } from "../std";
+
+export const SIM_CONTAINER_FQN = fqnForType("sim.Container");
+export const HOST_PORT_ATTR = "host_port";
+
+/**
+ * Initialization properties for `sim.Container`.
+ */
+export interface ContainerProps {
```

Comment:
Im curious about 2 other props and whether they should be included, `restart` and `volumes`. I think its pretty trivial to allow sepcifing the restart option.

Since this is a sim only implementation, allowing `volumes` might be fine.

Reply:
I wanted to keep this PR to a minimum and then we will can add more features later. ok?

Reply:
Eventually we might convert this to use `docker compose` and expose all of its features